### DISCUSSION TITLE
Need clarification

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spam-Lists#45] [General]
Are the SpamThreads.txt, the commenters who reply to a investment scam, or are they only names of people mentioned in those comments?

**Comment by KendallDoesCoding** [comment]
I think this is for the names which are mentioned in the comments right @ThioJoe ?

**Reply by ThioJoe** [reply]
Yes it is only for the names mentioned within threads

**Reply by KendallDoesCoding** [reply]
Ok, good to know!
